tests/playwright/sanity/modules/tabs-v2/tabs.test.js

```diff
@@ -0,0 +1,33 @@
+const { test, expect } = require( '@playwright/test' );
```

Comment:
@hein-obox @TzviRabinovitch Do we really need a PlayWright test for this task?

Reply:
@louiswol94 Test removed